pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -256,6 +256,7 @@ func (p *cadvisorStatsProvider) ImageFsStats() (*statsapi.FsStats, error) {
 
 // ImageFsDevice returns name of the device where the image filesystem locates,
 // e.g. /dev/sda1.
+// TODO: support multiple image FS
```

Comment:
suggest using issue to track instead of TODOs in the code

Reply:
Arktos issue 350 is created to track this work.